python/GafferSceneUI/CameraUI.py

```diff
@@ -152,8 +338,7 @@
 
 			"description",
 			"""
-			The horizontal field of view, specified in degrees.  When using a perspective projection
-			with "Field Of View" control mode, this will control the focal length.
+			The horizontal field of view, in degrees.
```

Comment:
I removed the extra detail because it was irrelevant to users. To see that plug, _Field of View_ mode has to already be active.

Now, from a reference perspective, you're right. I was doing my best to shorten all tooltips for the users.

Reply:
I guess this just wasn't the longest tooltip to start with?

Reply:
> I removed the extra detail because it was irrelevant to users.

Let's be clear about something : the description metadata is not just used as a tooltip, it is also used for the reference documentation. We shouldn't be "improving" the former at the expense of the latter. I don't think that one extra piece of information in an already tiny description is hurting, and "the users" use the reference documentation too.